source/contracts/reporting/Market.sol

```diff
@@ -377,7 +390,7 @@ contract Market is DelegationTarget, Typed, Initializable, Ownable, IMarket {
     }
 
     function getFinalWinningStakeToken() public view returns (IStakeToken) {
-        return stakeTokens[finalPayoutDistributionHash];
+        return IStakeToken(stakeTokens.getAsAddressOrZero(finalPayoutDistributionHash));
```

Comment:
It feels like this function should throw if there is no token at `finalPayoutDistributionHash`.  Should never be possible I suppose, so I guess doesn't really matter...